controllers/selfnoderemediationconfig_controller.go

```diff
@@ -190,3 +198,54 @@ func (r *SelfNodeRemediationConfigReconciler) syncCerts(cr *selfnoderemediationv
 	}
 	return nil
 }
+
+func (r *SelfNodeRemediationConfigReconciler) updateDsTolerations(objs []*unstructured.Unstructured, tolerations []corev1.Toleration) error {
+	r.Log.Info("Updating DS tolerations")
+	if len(tolerations) == 0 {
```

Comment:
Can tolerations be nil since this field is [`omitempty`](https://github.com/medik8s/self-node-remediation/pull/130/files#diff-792340dfdee839c7939f417270ed1aa40b9f59b0e2d757b5d5417f7a59fbeb9fR118)? If so , then we should check it first.

Reply:
this check covers nil as well.
i.e:
```
len(nil) is 0
```

Reply:
Not sure about `can't` but it definitely `shouldn't` be empty